shared/domains/groupIncome/contracts/index.js

```diff
@@ -0,0 +1,5 @@
+import INVITE from './invite'
+
+export default Object.assign({},
+  INVITE
+)
```

Comment:
This is weird, what's going on here?

If this is really necessary, can you add a comment explaining why?

Reply:
I want to keep different contracts in separate files, to make it easier to maintain them. and as they are the same domain, I will need to merge them into a single object when exporting. but right now I only have this one contract file, so this is not necessary, removing for now

Reply:
Ah I see, ok. This is fine, maybe just add a comment to the code to explain this, as it's just a little non-standard looking and might confuse people who stumble across it